learn.qmd

```diff
@@ -1,13 +1,82 @@
 ---
-title: "Learn"
-listing:
-  contents: learn
-  sort: "date desc"
-  type: default
-  categories: true
-  sort-ui: false
-  filter-ui: false
-  feed: true
-page-layout: full
-title-block-banner: true
+format:
+  html:
+    toc: true
 ---
+
+# Learning materials
+
+Our trainings (1) facilitate how to find, choose, and use
+R packages for Outbreak Analytics tasks and (2) extending our R toolset
+to produce open, reproducible and sustainable data analysis projects
+following good Research Sofware practices. These two goals constitute
+the **Outbreak Analytics curriculum** and the **Research Sofware
+curriculum**.
+
+We address these goals with an e-Epi training kit, Tutorials,
+How-to guides, and Case studies:
+
+- The **e-Epi training kit** is an e-learning strategy for progressive
+  and flexible learning. It includes an introductory e-course in data
+  science for public health and infectious disease modelling in Latin
+  America and the Caribbean with a gender perspective. Initially only
+  with content in Spanish. (coming soon)
+- **Tutorials** will help you learn how to complete an outbreak analytic
+  task following a practical step-by-step guide using R packages to
+  start your own journey.
+- **Case studies** are real-life retrospective accounts of real projects
+  delivered by users in the field. (coming soon)
+- **How-to guides** are question-specific recipe-like scripts to show
+  how multiple R packages can interoperate to get a solution.
+
+Please note the development stage of the learning materials. A
+"pre-alpha" tutorial is still being constructed. An
+"alpha" tutorial is being taught by the original authors but
+has not been fully tested.
+
+## Outbreak Analytics Curriculum
+
+These workshops focus on working with outbreak data using peer-reviewed interoperable open-source software like R packages to help the decision-making process in public health.
+
+### e-Epi training kit en Español
+
+El **e-Epi-training kit** es una estrategia para el aprendizaje en línea que permite un aprendizaje progresivo y flexible. Este kit incluye el curso virtual en ciencia de datos en salud pública y modelamiento de enfermedades infecciosas, el cual está dirigido a América Latina y el Caribe y cuenta con un enfoque de género. [Leer más detalles sobre la estrategia en línea](https://epiverse-trace.github.io/epi-training-kit/).
+
+### Tutoriales en Español
+
+| Tutorial                                                                                                                              | Repositorio                                                                    | Agenda para instructoras/es                                                                                  | Fase de desarrollo                                                                                                                                                   |
```

Comment:
@chartgerink could we keep isolated the name and site columns? replaced in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/157/commits/dba3bd49d11faa0306719480c2c11439173eec57 but replacing the long links with the icon

the `name | site | repo | agenda | phase` outlook seems to keep all columns in all tables with similar widths.